gui/gm-editor.lua

```diff
@@ -9,6 +9,18 @@ local utils = require 'utils'
 
 config = config or json.open('dfhack-config/gm-editor.json')
 
+function save_config(data)
+    for k,v in pairs(data) do
+        config.data[k] = v
+    end
+    config:write()
+end
+
+read_only = true
```

Comment:
I think an editor shouldn't start in read-only mode by default.
```suggestion
read_only = false
```